build.sbt

```diff
@@ -32,7 +29,7 @@ libraryDependencies ++= Seq(
 libraryDependencies ++= Seq(
   "org.mockito"              % "mockito-core"             % "1.9.5"  % "test",
   "org.scalacheck"          %% "scalacheck"               % "1.11.6" % "test",
-  "org.scalatest"           %% "scalatest"                % "2.2.1"  % "test"
+  "org.scalatest"           %% "scalatest"                % "2.2.4"  % "test"
```

Comment:
scalatest dependency is included in sbtplugins now, we can remove this line or make sure it's up to date.